iset.mm

```diff
@@ -60969,6 +61036,73 @@ the cancellation property (fifth hypothesis), show that the relation
       JUHAASTUAUBUC $.
   $}
 
+  ${
+    $d f x y z w A $.  $d f B $.  $d x y z w F $.
+    $( A version of ~ fnex for finite sets that does not require Replacement.
+       (Contributed by Mario Carneiro, 16-Nov-2014.)  (Revised by Mario
+       Carneiro, 24-Jun-2015.) $)
+    fnfi $p |- ( ( F Fn A /\ A e. Fin ) -> F e. Fin ) $=
+      ( vw vy vz cfn wcel wa cres cv c0 csn cun reseq2 eleq1d syl2anc cvv simpr
+      wceq cdm wfn fnresdm adantr res0 0fin eqeltri a1i wss resundi cfv simp-4l
+      cdif simplrr eldifad fnressn uneq2d wn elexd funfvex funfni opexg eldifbd
+      cop cin opeldmg dmres eleq2i syl6ib elin simplbi syl6 mtod unsnfi syl3anc
+      wi eqeltrd syl5eqel ex findcard2sd eqeltrrd ) BAUAZAFGZHZBAIZBFWAWDBSWBAB
+      UBUCWCBCJZIZFGBKIZFGZBDJZIZFGZBWIEJZLZMZIZFGZWDFGCDEAWEKSWFWGFWEKBNOWEWIS
+      WFWJFWEWIBNOWEWNSWFWOFWEWNBNOWEASWFWDFWEABNOWHWCWGKFBUDUEUFUGWCWIFGZHZWIA
+      UHZWLAWIULZGZHZHZWKWPXCWKHZWOWJBWMIZMZFBWIWMUIXDXFWJWLWLBUJZVCZLZMZFXDXEX
+      IWJXDWAWLAGZXEXISWAWBWQXBWKUKZXDWLAWIWRWSXAWKUMZUNZAWLBUOPUPXDWKXHQGZXHWJ
+      GZUQXJFGXCWKRXDWLQGXGQGZXOXDWLWTXMURXDWAXKXQXLXNXQAWLBWLBUSUTPZWLXGQQVAPX
+      DXPWLWIGZXDWLAWIXMVBXDXPWLWIBTZVDZGZXSXDXPWLWJTZGZYBXDXKXQXPYDVOXNXRWLXGW
+      JAQVEPYCYAWLBWIVFVGVHYBXSWLXTGWLWIXTVIVJVKVLWJXHQVMVNVPVQVRWAWBRVSVT $.
+  $}
+
+  $( The domain of a finite function is finite.  (Contributed by Jim Kingdon,
+     5-Feb-2022.) $)
+  fundmfi $p |- ( ( A e. Fin /\ Fun A ) -> dom A e. Fin ) $=
+    ( cfn wcel wfun cdm cen wbr fundmeng enfii syldan ) ABCADAEZAFGKBCABHKAIJ
+    $.
+
+  $( A function (set) is finite if and only if its domain is finite.
+     (Contributed by AV, 10-Jan-2020.) $)
+  fundmfibi $p |- ( Fun F -> ( F e. Fin <-> dom F e. Fin ) ) $=
+    ( wfun cfn wcel cdm fundmfi ancoms wfn funfn fnfi sylanb impbida ) ABZACDZA
+    EZCDZNMPAFGMAOHPNAIOAJKL $.
+
+  $( If a set is finite, its converse is as well.  (Contributed by Jim Kingdon,
+     5-Feb-2022.) $)
+  relcnvfi $p |- ( ( Rel A /\ A e. Fin ) -> `' A e. Fin ) $=
+    ( wrel cfn wcel wa ccnv cen wbr wceq dfrel2 biimpi adantr simpr eqeltrd cvv
+    relcnv cnvexg cnven sylancr adantl enfii syl2anc ) ABZACDZEZAFZFZCDUFUGGHZU
+    FCDUEUGACUCUGAIZUDUCUIAJKLUCUDMNUDUHUCUDUFBUFODUHAPACQUFORSTUFUGUAUB $.
+
+  $( The range of a finite relation is finite if its converse is a function.
+     (Contributed by Jim Kingdon, 5-Feb-2022.) $)
+  funrnfi $p |- ( ( Rel A /\ Fun `' A /\ A e. Fin ) -> ran A e. Fin ) $=
+    ( wrel ccnv wfun cfn wcel w3a crn cdm df-rn relcnvfi 3adant2 simp2 syl5eqel
+    fundmfi syl2anc ) ABZACZDZAEFZGZAHRIZEAJUAREFZSUBEFQTUCSAKLQSTMROPN $.
+
+  $( A 1-1 function (class) with a set as domain is finite if and only if its
```

Comment:
> An injective function whose domain is a set is finite if and only if its range is finite.  See also ~ f1vrnfibi .

? (ping @avekens )

Reply:
I will change this to "A one-to-one function whose domain is a set is finite if and only if its range is finite. See also ~ f1vrnfibi ." in set.mm. I think "one-to-one function" is more commonly used in set.mm than "injective function".

Reply:
I'll use the comment from #2475